plugins/inputs/ipmi_sensor/ipmi.go

```diff
@@ -146,12 +146,12 @@ func transform(s string) string {
 }
 
 func init() {
-	m := Ipmi{}
-	path, _ := exec.LookPath("ipmitool")
-	if len(path) > 0 {
-		m.Path = path
-	}
 	inputs.Add("ipmi_sensor", func() telegraf.Input {
+		m := Ipmi{}
+		path, _ := exec.LookPath("ipmitool")
+		if len(path) > 0 {
+			m.Path = path
+		}
```

Comment:
Not that `exec.LookPath` is expensive, and worth the effort to keep from repeating it, but you could do:

```
inputs.Add("ipmi_sensor", func() telegraf.Input {
  m := m
  return &m
}
```